microsoft-edge/web-platform/writing-assistance-apis.md

```diff
@@ -537,7 +537,7 @@ controller.abort();
 <!-- ====================================================================== -->
 ## Send feedback
 
-We're very interested in learning about the range of scenarios for which you intend to use the Writing Assistance APIs, any issues with the APIs or language models, and whether new APIs, such as APIs for proofreading would be useful.
+We're very interested in learning about the range of scenarios for which you intend to use the Writing Assistance APIs, any issues with the APIs or language models, and whether new APIs, such as APIs for proofreading, would be useful.
```

Comment:
"whether new task-specific APIs, such as for proofreading or translation, would be useful."